.gitignore

```diff
@@ -447,5 +447,5 @@ fabric.properties
 *.iml
 modules.xml
 
-### Ignore all .idea files
-**/.idea/
+# Rider files
```

Comment:
Nitpick: IDEA is the original JetBrains product which is used as an IDE for Java and Kotlin. The .idea folder is not specific for Rider, but for all JetBrains IDEs. If someone were to use PyCharm with a Rider plugin, this folder would still generate. Perhaps we can remove the dot, but is not specific to Rider files.